lib/application-stub.js

```diff
@@ -69,6 +69,15 @@
 			},
 
 			/**
+			 * Checks if a service exists
+			 * @param {string} serviceName The name of the service to check.
+			 * @returns {boolean} True, if service exist. False, otherwise.
+			 */
+			hasService: function(serviceName) {
+				return serviceName in services;
```

Comment:
I'd switch to `hasOwnProperty()` here. Right now, still will return `true` when `serviceName` is `toString`, `hasOwnProperty`, etc.